samples/CallingStateful/src/components/CallManager.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Call, TeamsCall } from '@azure/communication-calling';
+import { IStackStyles, PrimaryButton, Stack, Text, Theme, useTheme } from '@fluentui/react';
+import React, { useEffect, useState } from 'react';
+
+export interface CallManagerProps {
+  onSetResume: (call: Call | TeamsCall) => void;
+  calls: Call[] | TeamsCall[];
+  activeCall?: Call | TeamsCall;
+  onSetHold: (call: Call | TeamsCall) => void;
+  onEndCall: (call: Call | TeamsCall) => void;
+}
+
+export const CallManager = (props: CallManagerProps): JSX.Element => {
+  const { onSetResume, calls, onSetHold, activeCall } = props;
+  const theme = useTheme();
+  return (
+    <Stack verticalAlign={'start'} styles={callManagerContainerStyle(theme)}>
+      <Text variant="xLarge">Calls</Text>
+      <Stack>
+        {activeCall && (
+          <Stack>
+            <Text variant="large">Active Call</Text>
+            <CallItem call={activeCall} onSetResume={onSetResume} isActiveCall={true} onSetHold={onSetHold} />
+          </Stack>
+        )}
+        {
+          <Stack>
+            {calls && calls.length > 1 && <Text variant="large">Other Calls</Text>}
```

Comment:
this is brittle in the event `calls.length === 1 && call.id !== activeCall?.id` (which shouldn't ever happen but is an edge case.
Instead consider a filter first
```
const otherCalls = calls.filter(call => call.id !== activeCall?.id);
...
{otherCalls.length > 1 && <Text variant="large">Other Calls</Text>}
{otherCalls.map((call) => ... }
```